lib/guardian/utils.ex

```diff
@@ -3,7 +3,7 @@ defmodule Guardian.Utils do
   @doc false
   def stringify_keys(nil), do: %{}
   def stringify_keys(map) do
-    Enum.reduce(Dict.keys(map), %{}, fn(k,acc) -> Dict.put(acc, to_string(k), map[k]) end)
+    Enum.reduce(Map.keys(map), %{}, fn(k,acc) -> Map.put(acc, to_string(k), map[k]) end)
```

Comment:
I think for this case `Enum.reduce(map, %{}, fn({key, val},acc) -> Map.put(acc, to_string(key), val) end)` would be way better.